test/e2e/main_test.go

```diff
@@ -27,7 +27,14 @@ func TestMain(m *testing.M) {
 
 	testenv.Setup(
 		envfuncs.CreateCluster(kind.NewProvider(), kindClusterName),
-		envfuncs.CreateNamespace(namespace),
+		envfuncs.CreateNamespace(namespace, envfuncs.WithLabels(map[string]string{
+			"pod-security.kubernetes.io/enforce":         "restricted",
+			"pod-security.kubernetes.io/enforce-version": "latest",
+			"pod-security.kubernetes.io/audit":           "restricted",
+			"pod-security.kubernetes.io/audit-version":   "latest",
+			"pod-security.kubernetes.io/warn":            "restricted",
+			"pod-security.kubernetes.io/warn-version":    "latest",
```

Comment:
We can drop these lines since we're enforcing the profile. If something is not compliant we will be immediately blocked, hence there's no need for the warning directive. Finally, when something is not compliant we will get an error on the stderr, making the audit irrelevant